mPower2/mPower2/StudyBurstScheduleManager.swift

```diff
@@ -181,6 +202,20 @@ class StudyBurstScheduleManager : SBAScheduleManager {
     /// Subset of the finished schedules.
     public private(set) var finishedSchedules: [SBBScheduledActivity] = []
     
+    /// Subset of the past survey schedules that were not finished on the day they were scheduled.
+    public private(set) var pastSurveySchedules: [SBBScheduledActivity] = []
+    
+    /// The completion task to use for today.
+    public private(set) var todayCompletionTask: StudyBurstConfiguration.CompletionTask?
+    
+    /// The action bar item to display.
+    public private(set) var actionBarItem: TodayActionBarItem?
+    
+    /// Does the study burst have an active study burst for today?
```

Comment:
phrasing is a bit confusing--do you mean something like "Is a study burst active for today?" Or something else?